src/renderer/components/ft-sponsor-block-category/ft-sponsor-block-category.js

```diff
@@ -0,0 +1,174 @@
+import Vue from 'vue'
+import { mapActions } from 'vuex'
+import FtSelect from '../ft-select/ft-select.vue'
+
+export default Vue.extend({
+  name: 'FtSponsorBlockCategory',
+  components: {
+    'ft-select': FtSelect
+  },
+  props: {
+    categoryName: {
+      type: String,
+      required: true
+    }
+  },
+  data: function () {
+    return {
+      categoryColor: '',
+      skipOption: '',
+      skipValues: [
+        'Auto Skip',
+        // 'Prompt to skip',
+        'Show In Seek Bar',
+        'Don\'t Skip'
+      ],
+      colorValues: [
```

Comment:
I think having the same colors is a good idea but I also think that having the list of color repeating in multiple places isn't a good idea. Do you think a ft-color-selector component would be a good idea?

Reply:
I think it might be too much to create a component (I am not sure)
I do think we need to start reducing these kind of duplicate values that is supposed to be the same **all the time**

Reply:
Utils is now being used so these values won't be duplicated